### ISSUE TITLE
Retrieve just the 'meta' from a collection

### ISSUE THREAD

**Issue by bohde** [django-tastypie/django-tastypie#108]
It would be useful to have the option to retrieve the collection metadata without any records.

**Comment by SeanHayes** [comment]
I don't think this really maps well to any REST or HTTP standards. Probably the best thing to do would be to change the Paginators to obey limit=0, that way you'll get the "meta" returned with an empty "objects" list.
